src/main/java/monique/storage/Storage.java

```diff
@@ -0,0 +1,107 @@
+package monique.storage;
+
+import java.io.EOFException;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+
+import monique.task.Task;
+
+
+/**
+ * The <code>Storage</code> class handles the reading from and writing to a file that stores serialized
+ * <code>Task</code> objects.
+ * It provides methods to save and load tasks, and ensures that the file for storage exists.
+ */
+public class Storage {
+    private final String filePath;
+    private final File dbFile;
```

Comment:
Naming this variable dbFile might be slightly unfamiliar for some users who don't frequently see "db" in place of "database", could I suggest databaseFile or saveFile instead?